src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -318,6 +318,21 @@ public static class Items {
          * This tag defaults to {@link net.minecraft.world.item.Items#LAPIS_LAZULI} when not present in any datapacks, including forge client on vanilla server
          */
         public static final TagKey<Item> ENCHANTING_FUELS = neoforgeTag("enchanting_fuels");
+        /**
+         * Controls what items Piglins can use as default as a valid crossbow
+         * This tag defaults to {@link net.minecraft.world.item.Items#CROSSBOW} when not present in any datapacks, including forge client on vanilla server
```

Comment:
I don't recall if we still have tag defaulting logic, since I think optional tags were removed some time ago and never properly reimplemented...? I might be wrong on this. In any case, this work can be done at another time, since `enchanting_fuels` above has the same issue.